src/main/java/com/corndel/cashregister/Level4.java

```diff
@@ -10,6 +10,24 @@ public class Level4 {
    */
   public static boolean canMakeAmount(int target, List<Item> drawer) {
     // TODO
-    return false;
+
+    if (target <= 0) {
+      return true;
+    }
+
+    for (Item item : drawer) {
+        int itemValue = item.getValue();
```

Comment:
Great by using variables you are not unintentionally modifying the instance variables on the Items in the list!